process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
+
+* The DocuSign REST API location - `docusign.apilocation=${DOCUSIGN_APILOCATION:https://demo.docusign.net/restapi}`
+* The DocuSign account ID under which the app operates - `docusign.accountId=${DOCUSIGN_ACCOUNT_ID:}`
+* The App integration key from DocuSign (**Admin** > **API & Keys**) - `docusign.clientId=${DOCUSIGN_CLIENT_ID:}`
+* The DocuSign User GUID whom the app impersonates (**Admin** > **Users**) - `docusign.impersonatedUserGuid=${DOCUSIGN_IMPERSONATED_USER_ID:}`
+* The DocuSign OAuth server location - `docusign.authServer=${DOCUSIGN_AUTH_SERVER:account-d.docusign.com}`
+* The DocuSign JWT token lifetime - `docusign.jwtTokenExpirationInSeconds=${DOCUSIGN_JWT_LIFETIME:3600}`
+* The private RSA key of the DocuSign App for JWT authentication - `docusign.privateKey=${DOCUSIGN_RSA_KEY:}`
+
+The DocuSign default settings are:
+
+* Default email subject, of the email that DocuSign sends to the signer - `docusign.email.subject=${DOCUSIGN_DEFAULT_EMAIL_SUBJECT:Please sign this document}`
+* DocuSign **Sign Here** box, which is an invisible tab label - `docusign.signhere.label=${DOCUSIGN_DEFAULT_SIGNHERE_LABEL:SignHereTab}`
+* Default page in the document where the DocuSign **Sign Here** box should go - `docusign.signhere.page=${DOCUSIGN_DEFAULT_SIGNHERE_PAGE:1}`
+* Default X position in the page for the **Sign Here** box - `docusign.signhere.posx=${DOCUSIGN_DEFAULT_SIGNHERE_POSX:1}`
+* Default Y position in the page for the **Sign Here** box - `docusign.signhere.posy=${DOCUSIGN_DEFAULT_SIGNHERE_POSY:1}`
+
+The connector uses a stream mechanism to send and receive information between itself and Process Automation. The following properties are used to identify the connector actions:
+
+```typescript
+spring.cloud.stream.bindings.sendForSignatureConsumer.destination=docusignconnector.SEND_FOR_SIGNATURE
+spring.cloud.stream.bindings.downloadDocumentConsumer.destination=${docusign.connector.name}.DOWNLOAD_DOCUMENT
+```
+
+The name of the channel must match the implementation value defined in the service task which is formed as part of the BPMN definition.
+
+### BPMN tasks configuration
 
-The **SIGNDOCUMENT** action is used by the DocuSign connector to request a digital signature on a document.
+This [process definition](https://github.com/Alfresco/alfresco-process-connector-services/blob/develop/alfresco-process-docusign-connector-spring-boot-starter/docuSignProcess.bpmn20.xml){:target="_blank"} example shows how to set up the connector in Process Automation.
 
-The input parameters to request a DocuSign signature are:
+As part of the BPMN definition process, any service task that is responsible for sending the document needs to set the `docusignconnector.SEND_FOR_SIGNATURE` or `docusignconnector.DOWNLOAD_DOCUMENT` properties as the value for its implementation attribute.
+
+The following input parameters must be provided for DocuSign API in the Service task depending on the implementation:
+
```

Comment:
Can we do something similar to the rest of connectors? (https://docs.alfresco.com/process-automation/latest/model/connectors/rest/). For example: 

```
The actions that can be executed using the DocuSing connector are:

* SEND_FOR_SIGNATURE
* DOWNLOAD_DOCUMENT
```